skimage/transform/histogram_matching.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+
+
+def _get_separate_channels(image):
```

Comment:
Having this as a separate function may be overkill, if you can run something like `array(..., ndmin=3)` on any input, and then proceed from there.

Reply:
Got rid of the `_get_separate_channels`. 
Had to change the order of transposition, since `ndim` prepends the shape with ones (so transposition is no longer needed for these arrays).